src/snowflake/connector/pandas_tools.py

```diff
@@ -362,6 +362,11 @@ def drop_object(name: str, object_type: str) -> None:
         logger.debug(f"dropping {object_type} with '{drop_sql}'")
         cursor.execute(drop_sql, _is_internal=True)
 
+    def truncate_table(name: str) -> None:
```

Comment:
Since this function is only called from 1 spot then let's not declare it as a helper method.